src/main/resources/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/parameters/Base64EncodedStringParameterValue/Base64EncodedStringParameterValue.jelly

```diff
@@ -27,6 +27,9 @@ THE SOFTWARE.
         xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form"
         xmlns:i="jelly:fmt" xmlns:p="/lib/hudson/project">
         <f:entry title="${it.name}" description="${it.description}">
+            <div name="parameter" description="${it.description}">
```

Comment:
Hi, thank you for the comments.

For Q1: I simply followed the convention from Rebuild plugin. All Jelly files for different types of parameters are under [this folder] (https://github.com/jenkinsci/rebuild-plugin/tree/master/src/main/resources/com/sonyericsson/rebuild/RebuildAction). I think by following the convention, this name is also more self-explainable. if you insist. I will change the name back to *value.jelly*.

For Q2: I understand your concern. But from Rebuild plugin's point, it might be impossible to provide a generic jelly file because only the project that defines the customized parameter knows how it should be displayed.

Maybe the following proposal could solve the problem?
We have two Jelly file (instead one) for the two different purposes. E.g. *value.jelly* displays Gerrit Trigger's own objects. The other one (*Base64EncodedStringParameterValue.jelly*) helps Rebuild renders the page properly?
I think in this way, we can solve the conflict and the strong dependency issue.